tools/test-npm.sh

```diff
@@ -3,11 +3,11 @@
 set -e
 
 # always change the working directory to the project's root directory
-cd $(dirname $0)/..
+cd $(dirname ${BASH_SOURCE[0]})/..
```

Comment:
Hmm, what issue were you trying to solve here exactly? Maybe there's an alternative.